cloud/google/ssh.go

```diff
@@ -64,21 +64,26 @@ func cleanupSshKeyPairs() {
 }
 
 // It creates secret to store private key.
-func (gce *GCEClient) setupSSHAccess(m *clusterv1.Machine) error {
+func (gce *GCEClient) setupSSHAccess(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	// Create public/private key pairs
 	err := createSshKeyPairs()
 	if err != nil {
 		return err
 	}
 
-	config, err := gce.providerconfig(m.Spec.ProviderConfig)
+	machineConfig, err := gce.machineproviderconfig(machine.Spec.ProviderConfig)
 	if err != nil {
 		return err
 	}
 
-	err = run("gcloud", "compute", "instances", "add-metadata", m.Name,
+	clusterConfig, err := gce.clusterproviderconfig(cluster.Spec.ProviderConfig)
```

Comment:
I can't find this method "clusterproviderconfig()" in the PR. Is there a missing file add?

Reply:
I found it now!